managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -401,8 +401,6 @@ public void operationFailed(MetaStoreException e) {
         });
 
         scheduleTimeoutTask();
-
-        scheduleRollOverLedgerTask();
```

Comment:
Just a question:
It will be executed regularly, why is it “the scheduled task will not be re-executed” ？

Reply:
It seems that the fixed-rate scheduled task doesn't roll over the ledger in time when the ledger reaches the maximum rollover threshold.

Reply:
We have now deleted the scheduled task. If there is no traffic, will it never be cleaned up?

Reply:
If there is no traffic the old task `scheduleRollOverLedgerTask` will also do nothing, because there is an entry num check before the rollover task.